Document LastCompletionResult isn't available to Scheduled WF runs when coupled with CAN (#2455)
temporalio/documentation: docs-src/concepts/what-is-a-schedule.md

Failures and timeouts do not affect the last completion result.

:::note

> When a Schedule triggers a Workflow that completes successfully and yields a result, this result from the initial Schedule execution can be accessed by the subsequent scheduled execution through the `LastCompletionResult`.

Reviewer: ```suggestion
When a Schedule triggers a Workflow that completes successfully and yields a result, the result from the initial Schedule execution can be accessed by the subsequent scheduled execution through `LastCompletionResult`.
```
Per https://developers.google.com/style/code-in-text#grammatical-treatment-of-code-elements.

> It's important to be aware that if, during the second run, the Workflow employs the [Continue-As-New](/concepts/what-is-continue-as-new) function, the `LastCompletionResult` won't be accessible for this new Workflow iteration.

Reviewer: ```suggestion
Be aware that if, during the subsequent run, the Workflow employs the [Continue-As-New](/concepts/what-is-continue-as-new) feature, `LastCompletionResult` won't be accessible for this new Workflow iteration.
```

> It is important to note that the [status](/concepts/what-is-a-workflow-execution#status) of the second run is marked as `Continued-As-New`, and not as `Completed`.

Reviewer: ```suggestion
It is important to note that the [status](/concepts/what-is-a-workflow-execution#status) of the subsequent run is marked as `Continued-As-New` and not as `Completed`.
```
